src/plot_api/plot_api.js

```diff
@@ -2124,6 +2126,7 @@ Plotly.update = function update(gd, traceUpdate, layoutUpdate, traces) {
             data: restyleSpecs.eventData,
             layout: relayoutSpecs.eventData
         });
+        gd.emit('plotly_plotmodified');
```

Comment:
Do you think it should be a separate event for this purpose? Again, my fear was coupling the component update code with events that serve other purposes. But I can try to use `plotly_afterplot` if you prefer. (though, again, it would need to be called during animation)